devito/ir/iet/visitors.py

```diff
@@ -197,7 +208,7 @@ def _gen_struct_decl(self, obj, masked=()):
             while issubclass(ctype, ctypes._Pointer):
                 ctype = ctype._type_
 
-            if not issubclass(ctype, ctypes.Structure):
+            if not issubclass(ctype, ctypes.Structure) or issubclass(ctype, NoDeclStruct):
```

Comment:
lines longer than 84-85 should break (I need to update the coding guidelines but I've been enforcing this for a while)